### PR TITLE
Detail view button issue(20 May)

### PR DESCRIPTION
**Pull request by Palakagrawal28**

### CODE REVIEW

File: src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx [hunk 1]
```diff
@@ -100,11 +100,11 @@ class MemberDetails extends React.Component {
         <div key={memberDetails.id}>
           {/*{TODO we need to break down it into components with dynamic values currently it's not configurable}*/}
           <div className="mbr-detail-header" onClick={this.props.onClickCloseModal}>
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx:102]:
```suggestion
          <div className="member-details-view-route__header" onClick={this.props.onClickCloseModal}>
```

File: src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx [hunk 2]
```diff
@@ -100,11 +100,11 @@ class MemberDetails extends React.Component {
...
-            <Link to={Routes.MEMBER_LIST_VIEW}>
-              <span>
-                <IconArrowLeft />
-              </span>
-            </Link>
+            <div className="mbr-detail-header__left">
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx:103]:
```suggestion
            <div className="member-details-view-route__header__left">
```

File: src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx (hunk 1 above)
Comment (Palakagrawal28) [reply src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx:102]: Done

File: src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx (hunk 2 above)
Comment (Palakagrawal28) [reply src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx:103]: Done

File: src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx (hunk 1 above)
Comment (mohitsaladiya) [reply src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx:102]: are there any change required in scss file. If the className is changed?

File: src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx (hunk 2 above)
Comment (mohitsaladiya) [reply src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx:103]: are there any change required in scss file. If the className is changed?

**Pull request merged**